ProjectWithUMLDiagram/src/com/beyond/person/core/service/interfaces/DancerAction.java

```diff
@@ -0,0 +1,5 @@
+package com.beyond.person.core.service.interfaces;
```

Comment:
@AraChilingaryan never create interfaces package , interfaces should be in service package

Reply:
done